recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
Should a conda package for libdrmaa be created and this package depend on it?  If a user installed this package without libdrmaa installed on their system it wouldn't work would it.